README.md

```diff
@@ -115,7 +110,7 @@ The included `benchmark_runner.py` script will run all queries sequentially. Con
 To run all queries, cd to `tpcx_bb/` and:
 
 ```python
-python benchmark_runner.py
+python benchmark_runner.py --config_file benchmark_runner/benchmark_config.yaml
 ```
 
 By default, this will run each query three times. You can control the number of repeats by changing the `n_repeats` variable in the script.
```

Comment:
Looks like we changed this default to run only once, and updated the variable to use all caps since it's a constant
```suggestion
By default, this will run each query once. You can control the number of repeats by changing the `N_REPEATS` variable in the script.
```